ssl/record/rec_layer_s3.c

```diff
@@ -917,7 +917,7 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
             SSL3_RECORD_add_length(thiswr, 1);
 
             /* Add TLS1.3 padding */
-            max_send_fragment = ssl_get_max_send_fragment(s);
+            max_send_fragment = ssl_get_max_send_fragment(s) + 1;
```

Comment:
If you want, I can make this +/-1 change a separate PR.
Note: I have no specific test cases where this would make a difference.